1.Exploratory_Data_Analysis/README.md

```diff
@@ -0,0 +1,7 @@
+# This folder hosts jupyter notebooks for exploratory data analysis (EDA) for multiple 'waves' of data. See this repo's main page for a link to the Interstellar preprocessing page.
```

Comment:
This may be defined else where. If not, considered providing a definition about what `waves` means for contextual purposes.